crates/jit/src/profiling.rs

```diff
@@ -20,7 +20,7 @@ cfg_if::cfg_if! {
 }
 
 cfg_if::cfg_if! {
-    if #[cfg(target_os = "linux")] {
+    if #[cfg(unix)] {
```

Comment:
Can you also change this in cranelift/jit?

Reply:
done!

Reply:
Thanks!